docs/faststore/docs/api-extensions/extending-api-schema.mdx

```diff
@@ -1,29 +1,24 @@
----
-title: 'Extending API schemas'
-sidebar_label: 'API Extensions'
-slug: extending-api-schema
----
+# Extending API schemas
 
-In this guide, learn how to extend VTEX and third-party APIs to customize them according to your store's needs, allowing you to efficiently retrieve and manipulate the necessary data.
+## Before you begin
 
----
+<Steps>
+### Ensure consistency with API extension best practices
 
-## Before you begin
+Even though you can add information to the FastStore API schema, you must be careful not to over-fetch data on your pages. Check the [best practices for fetching data on your storefront](https://developers.vtex.com/docs/guides/faststore/api-extensions-overview#best-practices-for-fetching-data).
```

Comment:
```suggestion
When extending the FastStore API schema, avoid overfetching data on your storefront. Refer to the [Best practices for fetching data on your storefront](https://developers.vtex.com/docs/guides/faststore/api-extensions-overview#best-practices-for-fetching-data) for more information.
```

Reply:
Essa frase pode ser mais direta: "Even though you can add information to the FastStore API schema, you must be careful not to"